stdeb/downloader.py

```diff
@@ -49,51 +37,20 @@ def find_tar_gz(package_name, pypi_url='https://pypi.org',
                              'releases %r' % (release, all_releases))
         version = release
     else:
-        default_releases = _call(pypi.package_releases, package_name)
-        if len(default_releases) != 1:
-            raise RuntimeError('Expected one and only one release. '
-                               'Non-hidden: %r. All: %r' %
-                               (default_releases, all_releases))
-        default_release = default_releases[0]
+        default_release = pypi_client.release_versions(package_name, current_only=True)
         if verbose >= 2:
             myprint(
-                'found default release: %s' % (', '.join(default_releases),))
+                'found default release: %s' % (', '.join(default_release),))
```

Comment:
I think this is actually a long-lived typo and the intent was always to log the single default release found.